db-scripts/src/main/resources/migration.sql

```diff
@@ -549,3 +549,42 @@ CREATE TABLE `cancer_study_tags` (
 );
 
 UPDATE `info` SET `DB_SCHEMA_VERSION`="2.7.3";
+
+##version: 2.7.4
+ALTER TABLE mutation 
+    ADD COLUMN DIP_LOG_R FLOAT,
+    ADD COLUMN CELLULAR_FRACTION FLOAT,
+    ADD COLUMN TOTAL_COPY_NUMBER INTEGER(6),
+    ADD COLUMN MINOR_COPY_NUMBER INTEGER(6),
+    ADD COLUMN CELLULAR_FRACTION_EM FLOAT,
+    ADD COLUMN PURITY FLOAT,
+    ADD COLUMN PLOIDY FLOAT,
+    ADD COLUMN CCF_M_COPIES FLOAT,
+    ADD COLUMN CCF_M_COPIES_LOWER FLOAT,
+    ADD COLUMN CCF_M_COPIES_UPPER FLOAT,
+    ADD COLUMN CCF_M_COPIES_PROB95 FLOAT,
+    ADD COLUMN CCF_M_COPIES_PROB90 FLOAT,
+    ADD COLUMN CCF_M_COPIES_EM FLOAT,
+    ADD COLUMN CCF_M_COPIES_LOWER_EM FLOAT,
+    ADD COLUMN CCF_M_COPIES_UPPER_EM FLOAT,
+    ADD COLUMN CCF_M_COPIES_PROB95_EM FLOAT,
+    ADD COLUMN CCF_M_COPIES_PROB90_EM FLOAT,
+    ADD COLUMN TOTAL_COPY_NUMBER_EM INTEGER(6),
+    ADD COLUMN MINOR_COPY_NUMBER_EM INTEGER(6);
+
+ALTER TABLE copy_number_seg 
```

Comment:
preparing for future facets features (adding additional info to segment data)
we must test that there is proper function of seg data before we merge this PR to master. Otherwise, we may need to drop this or expand the persistence layer and model